subsys/net/l2/canbus/canbus.c

```diff
@@ -886,6 +938,14 @@ static inline int send_ff(struct net_pkt *pkt, size_t len, bool mcast,
 		index += lladdr_inline->len;
 	}
 
+#ifdef CONFIG_NET_L2_CANBUS_ETH_TRANSLATOR
+	if (net_pkt_lladdr_src(pkt)->type == NET_LINK_ETHERNET) {
```

Comment:
Please remove the ifdef and put IS_ENABLED(CONFIG_NET_L2_CANBUS_ETH_TRANSLATOR) info if statement.